pkg/plan/export.go

```diff
@@ -60,6 +66,27 @@ func (p *Plan) Export(ctx context.Context) error {
 	return helper.SaveInterface(ctx, p.fullPath, p.body)
 }
 
+func (p *Plan) removeUnchanged() {
+	filtered := p.body.Releases[:0]
+
+	isChanged := func(r release.Config) bool {
+		for _, uc := range p.unchanged {
+			if r.Uniq().Equal(uc) {
+				return false
+			}
+		}
+		return true
+	}
```

Comment:
You can use existing helper for that :)
https://github.com/helmwave/helmwave/blob/main/pkg/helper/in.go#L9

Reply:
But it will require to change `unchanged` slice from `[]uniqname.UniqName` to `[]release.Config`. It won't bump memory consumption because instead of storing list of strings you will store list of pointers

Reply:
Ok. I'll do it soon